src/components/atomic-swap/ModalAtomicReceive.js

```diff
@@ -116,9 +115,7 @@ export function ModalAtomicReceive ({ sendClickHandler, receivableTokens, manage
                                 <label htmlFor="amount" className="col-3 my-auto">{t`Amount`}: </label>
                                 <InputNumber key="value"
                                              name="amount"
-                                             ref={amountRef}
-                                             defaultValue={hathorLib.numberUtils.prettyValue(amount, decimalPlaces)}
-                                             placeholder={hathorLib.numberUtils.prettyValue(0, decimalPlaces)}
+                                             defaultValue={amount}
```

Comment:
The HTML `<input>` element's `value` is a `string` — as it was in `master`, so it supports separators (e.g. `1,000.00`).

The `InputNumber` component's `defaultValue` prop and `value` state are `Number`, so all handling is done with (whole) numbers. Then, the `value` state is converted to a `string` with `format` so it can be used in the HTML element's `value`.